### DISCUSSION TITLE
How to customize validation messages?

### DISCUSSION THREAD

**Discussion by iamlasse** [filamentphp/filament#1281] [General]
https://github.com/laravel-filament/filament/blob/0ddd6c8d6a9d3be13ec23c55a94b4581812a92c6/packages/forms/src/Concerns/CanBeValidated.php#L81

**Comment by danharrin** [comment]
Maybe you could use the method from the Livewire docs?

https://laravel-livewire.com/docs/2.x/input-validation#customize-error-message-and-attributes

If you're using the admin panel, these messages need to go on the Page classes (Create and Edit) and not the Resource class, and each field name needs to be prefixed with `data.`.

**Reply by iamlasse** [reply]
I was using the data property and missed it in my messages lol Thanks for quick response!

**Reply by untillnesss** [reply]
@danharrin What if I want to provide a custom error message in the Create Action Dialog, not in Create Page?

**Reply by danharrin** [reply]
Probably prefix with mountedActionData.0.
